fbg-server/src/rooms/rooms.service.ts

```diff
@@ -35,37 +37,41 @@ export class RoomsService {
     roomEntity.capacity = room.capacity;
     roomEntity.gameCode = room.gameCode;
     roomEntity.isPublic = room.isPublic;
+    const user = await this.usersService.getUserEntity(userId);
     if (!queryRunner) {
       await inTransaction(this.connection, async (queryRunner) => {
-        await this.saveNewRoom(queryRunner, userId, roomEntity);
+        await this.saveNewRoom(queryRunner, user, roomEntity);
       });
       if (room.isPublic) {
         // Cant do this inside a transaction as we need the new room in the notification.
         await this.lobbyService.notifyLobbyUpdate();
+        const doNothing = () => { return; };
+        this.maybeNotifyDiscord(user, roomEntity).then(doNothing, doNothing); // Fire and forget
```

Comment:
I'm just curious, why is the .then(doNothing, doNothing) necessary? What's the difference between this and ignoring the returned promise?

Reply:
You were right, this is not necessary.